### ISSUE TITLE
Add support for Gitlab

### ISSUE THREAD

**Issue by peterjaap** [shaneosullivan/gitmeme#19]
Is it possible to add support for Gitlab (gitlab.com and self-hotsed instances)?

Because, if so;

<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia2.giphy.com%2Fmedia%2Fl0MYt5jPR6QX5pnqM%2Fgiphy.gif&token=party" data-gitmeme-token="party"><img src="https://media2.giphy.com/media/l0MYt5jPR6QX5pnqM/giphy.gif" title="Created by gitme.me with /party"/></a>

**Comment by abstractlyZach** [comment]
Agreed! It would be great if we could add different url patterns to this list:
![Screen Shot 2021-05-24 at 1 43 58 PM](https://user-images.githubusercontent.com/7566043/119405553-498c9b80-bc96-11eb-9ec8-bf629b5fc497.png)
